lib/documentLoader.js

```diff
@@ -1,12 +1,23 @@
 /*!
  * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
  */
+import * as credentialsV2Context from '@digitalbazaar/credentials-v2-context';
 import dataIntegrityContext from '@digitalbazaar/data-integrity-context';
+import multikeyCtx from '@digitalbazaar/multikey-context';
 import {securityLoader} from '@digitalbazaar/security-document-loader';
 
 export const loader = securityLoader();
 
-loader.addStatic(
-  dataIntegrityContext.constants.CONTEXT_URL,
-  dataIntegrityContext.contexts.get(
-    dataIntegrityContext.constants.CONTEXT_URL));
+const addContext = ({url, contexts}) =>
+  loader.addStatic(url, contexts.get(url));
```

Comment:
Is this `addContext` helper supposed to enumerate `contexts` or something? It doesn't seem to do anything more than what `loader.addStatic` is already doing.

Is there more than one context from the DI context module that should be getting added here?

Reply:
just made the code look cleaner as you did have to fetch a context from a Map with a really long name. I think it improves readability.

Reply:
changed from `addContext` to `addContexts` here: https://github.com/digitalbazaar/vc-holder-http/pull/4/commits/5c99d73b55ce9122e3fe4cfffe11db8915258944

It now adds all values in the contexts map.